deployment/upgradeLive.sh

```diff
@@ -21,6 +20,7 @@ if [ ! -d "$1" ] ; then
         unlink /opt/kaltura/liveController/latest
         ln -s /opt/kaltura/liveController/$1 /opt/kaltura/liveController/latest
```

Comment:
I think creation of symbolic link should be at the end of the script after everything has passed properly and before restarting process. 
That way, if something fails it will not overwrite the last version that was linked and was working.